opendevin/sandbox/docker/README.md

```diff
@@ -0,0 +1,395 @@
+# Use SWE-Bench Environment in the OpenDevin Sandbox
+
+This README provides guidance on how to use the SWE-Bench environment within the OpenDevin sandbox. Note that this integration is actively being developed and may undergo changes.
+
+Warning: we are in the process of dockerizing SWE-bench utility files, which are currently prepared and managed externally.
+
+## Prerequisites
+
+### Prepare SWE-Bench utilities
+
+To set up, please utilize our modified version at [OD-SWE-bench/tree/eval](https://github.com/OpenDevin/OD-SWE-bench/tree/eval). Follow the [instructions](https://github.com/OpenDevin/OD-SWE-bench/blob/eval/swebench/harness/OD_README.md) to prepare both the testbeds and necessary conda environments for OD-SWE-Bench and SWE-Bench test instances.
```

Comment:
It will be good if we can provide a block of bash command where people can just run it to get everything done